servicecontrol/plugins.md

```diff
@@ -89,6 +89,10 @@ Note that even if an endpoint is able to send heartbeat messages and it is marke
 If a heartbeat message is not received by ServiceControl from an endpoint, that endpoint is marked as "inactive". 
 An inactive endpoint indicates that there is a failure in the communication path between ServiceControl and the monitored endpoint. For example, such failures may be caused by a failure of the endpoint itself, a communication failure in the transport, or when ServiceControl is unable to receive and process the heartbeat messages sent by the endpoint.
 
+
+#### Should I deploy the Heartbeat plugin to Production?
+Please ensure that you **do deploy** this plugin to your endpoint in production. The information sent by this plugin is vital for monitoring. ServicePulse will not be able to indicate if your endpoint is active or not without this information.
```

Comment:
I added the section to all the plugins because:
it isn't clear which plugin you deploy and which one you don't 
I felt every plugin needed a section that made it very clear to the user on do I need this or not. 
I am happy to hear your suggestions on how you'd re-word it.